code/html/index.html

```diff
@@ -0,0 +1,64 @@
+          <td><input type="button" name="nameEight" class="rcalc-button" value="8" onclick="testZero(nameEight.value);displayNumber(nameEight.value);"></td>
+          <td><input type="button" name="nameNine" class="rcalc-button" value="9" onclick="testZero(nameNine.value);displayNumber(nameNine.value);"></td>
+          <td><input type="button" name="nameDivision" class="rcalc-button" value="/" style="background-color: #39681c" onclick="setWarning();displayNumber(nameDivision.value);"></td>
+        </tr>
+        <tr>
+          <td><input type="button" name="nameFour" class="rcalc-button" value="4" onclick="testZero(nameFour.value);displayNumber(nameFour.value);"></td>
+          <td><input type="button" name="nameFive" class="rcalc-button" value="5" onclick="testZero(nameFive.value);displayNumber(nameFive.value);"></td>
+          <td><input type="button" name="nameSix" class="rcalc-button" value="6" onclick="testZero(nameSix.value);displayNumber(nameSix.value);"></td>
+          <td><input type="button" name="nameMultiplication" class="rcalc-button" value="*" style="background-color: #39681c" onclick="displayNumber(nameMultiplication.value);"></td>
+        </tr>
+        <tr>
+          <td><input type="button" name="nameOne" class="rcalc-button" value="1" onclick="testZero(nameOne.value);displayNumber(nameOne.value);"></td>
+          <td><input type="button" name="nameTwo" class="rcalc-button" value="2" onclick="testZero(nameTwo.value);displayNumber(nameTwo.value);"></td>
+          <td><input type="button" name="nameThree" class="rcalc-button" value="3" onclick="testZero(nameThree.value);displayNumber(nameThree.value);"></td>
+          <td><input type="button" name="nameSubtraction" class="rcalc-button" value="-" style="background-color: #39681c" onclick="displayNumber(nameSubtraction.value);"></td>
+        </tr>
+        <tr>
+          <td><input type="button" name="nameDot" class="rcalc-button" value="." onclick="displayNumber(nameDot.value);"></td>
+          <td><input type="button" name="nameZero" class="rcalc-button" value="0" onclick="testZero(nameZero.value);displayNumber(nameZero.value);"></td>
+          <td><input type="button" name="nameEqual" class="rcalc-button" value="=" onclick="result();"></td>
+          <td><input type="button" name="nameAddition" class="rcalc-button" value="+" style="background-color: #39681c" onclick="displayNumber(nameAddition.value);"></td>
+        </tr>
+        <tr>
+          <td colspan="4"><textarea name="nameDisplayInfo" id="id-display-info" rows="2" disabled></textarea></td>
+        </tr>
+      </table>
+    </form>
+    <br>
+    <div class="rcalc-dropdown">
+      <button class="rcalc-dropdown-button">Color</button>
+      <div class="rcalc-dropdown-content">
+        <button type="button" class="rcalc-button-css-swap" onclick="swapCssFile('D:/calculator/code/css/theme-default.css')">Default theme</button>
```

Comment:
you are correct, after creation of folder structure, Im having a trouble to refer to files to make it work anywhere else and not just in my PC

Reply:
Everybody has some problem but do it this things is not professional at all!!! You can't do it commit not working code!!!

Reply:
In some project exists money-box and in case of some error which brakes application you will put some money into money-box so be careful before create PR.